Terminal.Gui/Application/Application.Keyboard.cs

```diff
@@ -332,39 +332,71 @@ internal static void AddApplicationKeyBindings ()
                     Command.NextView,
                     () =>
                     {
-                        ApplicationNavigation.MoveNextView ();
-
-                        return true;
+                        View? current = Application.Current;
```

Comment:
No real reason to grab this reference. Just in-line it (`ctrl + r, i` by default in ReSharper with the caret on `current`). 

In fact, lines 335 through 340 can reduce all the way down to:

```csharp
return Application.Current?.AdvanceFocus (NavigationDirection.Forward, TabBehavior.TabStop)
```

Also, please make any anonymous function static that can be. It makes a pretty big difference in the JIT output.

If you start them off as `static () => {}` before you even write their bodies, it'll also help force you to avoid closure capture of locals, which is a nice perk.

Looks like this one can be static, since it doesn't reference anything that would disqualify it.

That results in the AddCommand call starting on line 331 turning into something like this:

```csharp
AddCommand (
            Command.NextView,
            static () => Application.Current?.AdvanceFocus (NavigationDirection.Forward, TabBehavior.TabStop));
```

Same simplification available with the rest, it look like.

Reply:
Actually,, they can all be static. Thanks.

Reply:
Nice